src/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -39,6 +39,11 @@
     <Configurations>Debug;Release;Proto</Configurations>
   </PropertyGroup>
 
+  <!--  The FSharp.Core dll provides a referencable public interface -->
+  <PropertyGroup Condition="'$(Configuration)' != 'Proto' and '$(CompressAllMetadata)' != 'true'">
```

Comment:
This doesn't look correct.  Shouldn't this be`'$(CompressAllMetadata)' == 'true'`